ui/pages/desktop-pairing/desktop-pairing.component.js

```diff
@@ -163,22 +171,22 @@ export default function DesktopPairingPage({
         >
           {t('done')}
         </Button>
-      </div>
+      </Box>
     );
   };
 
   return (
-    <div className="page-container__content">
-      <div className="desktop-pairing">
-        {renderIcon()}
-        <div className="desktop-pairing__title">{t('desktopPageTitle')}</div>
-        <div className="desktop-pairing__subtitle">
-          {t('desktopPageSubTitle')}
-        </div>
-      </div>
-      <div className="desktop-pairing">{renderContent()}</div>
+    <Box
+      display={DISPLAY.FLEX}
+      flexDirection="column"
+      alignItems={AlignItems.center}
+      marginLeft={2}
+      marginRight={2}
+    >
+      {renderIcon()}
+      {renderContent()}
       {renderFooter()}
-    </div>
+    </Box>
   );
 }
 
```

Comment:
Is there a way we could display this component in storybook using mocked redux state?

Create a file called `ui/pages/desktop-pairing/desktop-pairing.stories.js`

Then run 
```
yarn storybook
```

This won't work because I'm not exactly sure what the state is to show the otp but you might be able to adjust it?


```
import React from 'react';
import { Provider } from 'react-redux';

import testData from '../../../.storybook/test-data';
import configureStore from '../../store/store';

const otpData = {
  ...testData,
  metamask: { ...testData.metamask, otp: '123456' },
};

const store = configureStore(otpData);

import DesktopPairingPage from '.';

export default {
  title: 'Components/Pages/DesktopPairingPage',
  decorators: [(story) => <Provider store={store}>{story()}</Provider>],
  argTypes: {
    mostRecentOverviewPage: {
      controls: 'text',
    },
    showLoadingIndication: {
      action: 'showLoadingIndication',
    },
    hideLoadingIndication: {
      action: 'hideLoadingIndication',
    },
    generateDesktopOtp: {
      action: 'generateDesktopOtp',
    },
  },
  args: {},
};

export const DefaultStory = () => <DesktopPairingPage />;

DefaultStory.storyName = 'Default';
```

It should show up like this. Which will allow for visual review without having to build flask and it's accessible on a successful build of your PR through a storybook build link. Here I commented out lines `95 -  98` to show the content

<img width="1440" alt="Screenshot 2023-03-20 at 11 32 08 AM" src="https://user-images.githubusercontent.com/8112138/226434078-016f6c80-ba67-4ebb-9d42-21400d5e3637.png">

Reply:
This is also missing the actual OTP as well

Reply:
Added a working default story 👍